infrastructure/constructs/lambda_config.py

```diff
@@ -11,3 +16,9 @@
     RETENTION_DAYS = aws_logs.RetentionDays.ONE_YEAR
 else:
     RETENTION_DAYS = aws_logs.RetentionDays.THREE_MONTHS
+
+PACKAGING_DIR = tempfile.mkdtemp(dir=BACKEND_DIRECTORY, prefix=".lambda_out_")
+
+
+def delete_packaging_dir(packaging_dir: str = PACKAGING_DIR) -> None:
```

Comment:
This function is never called with `packaging_dir`, so we might as well remove the argument.